src/common/strtol.cc

```diff
@@ -16,6 +16,7 @@
 
 #include <climits>
 #include <limits>
+#include <math.h>
```

Comment:
Prefer C++ inclusion with <cmath>, because it will place things into the std:: namespace.